packages/kit/src/exports/public.d.ts

```diff
@@ -430,6 +430,31 @@ export interface KitConfig {
 		 */
 		errorTemplate?: string;
 	};
+	/**
+	 * Image optimization configuration
+	 */
+	images?: {
+		/**
+		 * Path to a a file that contains a loader that will be used to generate the an image URL out of the given source and width.
```

Comment:
```suggestion
		 * Path to a file that contains a loader that will be used to generate the an image URL out of the given source and width.
```

Reply:
Perhaps we should say "module" rather than "file"? It could be good to give examples of both a module from an npm package and local file being used here